lit/SwiftREPL/SwiftInterface.test

```diff
@@ -0,0 +1,17 @@
+// Test that .swiftinterface files can be loaded via the repl.
+
+// RUN: rm -rf %t
+// RUN: mkdir %t
+// RUN: %target-swiftc -module-name A -emit-parseable-module-interface-path %t/A.swiftinterface -emit-library -o %t/libA%target-shared-library-suffix %S/Inputs/A.swift
+// RUN: %lldb --repl="-I%t -L%t -lA -enable-parseable-module-interface" < %s | FileCheck %s
```

Comment:
It might be good to make sure that it is impossible to accidentally import the library from source code, perhaps by moving away the sources.